apps/webapp/app/presenters/RunListPresenter.server.ts

```diff
@@ -138,4 +133,27 @@ export class RunListPresenter {
       },
     };
   }
+
+  // we want to remove `null` from username
+  #getUsername(user?: {
```

Comment:
I think this is repeated in RunPresenter, it would be good to extract this out into a single function. Also, is there any backup in case this #getUsername returns undefined?

Reply:
`#getUsername` shouldn't return `null` because all the registered users will have a name set.
Either way, if it does, the username will not be appended to the env-label in RunList and Run pages (Dev: _username_ vs Dev) which should be fine I hope.